src/frontend/src/lib/components/transactions/AllTransactions.svelte

```diff
@@ -1,13 +1,21 @@
 <script lang="ts">
+	import NetworksSwitcher from '$lib/components/networks/NetworksSwitcher.svelte';
 	import AllTransactionsList from '$lib/components/transactions/AllTransactionsList.svelte';
 	import MessageBox from '$lib/components/ui/MessageBox.svelte';
 	import PageTitle from '$lib/components/ui/PageTitle.svelte';
+	import { testnetsEnabled } from '$lib/derived/settings.derived';
 	import { i18n } from '$lib/stores/i18n.store';
 </script>
 
 <div class="flex flex-col gap-5">
 	<PageTitle>{$i18n.activity.text.title}</PageTitle>
 
+	{#if $testnetsEnabled}
+		<div>
```

Comment:
You need this div?

Reply:
alas yes... otherwise it will use full width and to change it i need to put the div inside and make a prop

(unless you prefer)

Reply:
I did not get it. Figma screenshots explained it. 😅